aepsych/strategy.py

```diff
@@ -174,34 +174,41 @@ def normalize_inputs(self, x, y):
         """converts inputs into normalized format for this strategy
 
         Args:
-            x (np.ndarray): training inputs
-            y (np.ndarray): training outputs
+            x (torch.Tensor): training inputs
```

Comment:
Alongside doc changes, add typehints. This should cause many mypy errors to chase through and make sure everything is uniform.

Reply:
This one is still missing return typehints. Should be torch.Tensor, torch.Tensor, int.